src/test-helpers/exportWithoutRoot.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Remove root, de-indent (trim), and append newline to make tests cleaner.
+ */
+export const exportedWithoutRoot = exported => exported.slice(exported.indexOf('\n'))
```

Comment:
`removeRoot` would be a better name for this function.

Reply:
That makes sense

Reply:
@anmolarora1